app/views/auth/userDropdown.scala.html

```diff
@@ -8,9 +8,9 @@
   <a href="@routes.Pref.form("game-display")"@if(playing){ target="_blank"}><span class="text" data-icon="%">@trans.preferences()</span></a>
   <a href="@routes.Auth.logout"><span class="text" data-icon="w">@trans.logOut()</span></a>
   <span class="sep"></span>
-  <div class="status">
+  <a class="status" href="/lag">
     <div class="led"></div>
```

Comment:
call me old fashioned, but seeing `<div>` inside `<a>` makes me cringe. Can you make them `<span>`?